sinonjs/sinon
# Document spying on accessors

Closes #1606

 #### Purpose (TL;DR) - mandatory

Adds documentation for an undocumented feature.











 #### How to verify - mandatory
1. Check out this branch
2. Open `docs/_releases/latest/spies.md`

 #### Checklist for author
